CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ Here is an overview of all new **experimental** features:
 ### Improvements
 
 - **General**: TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
+- **Azure pipeline Scaler**: Add support for workload identity authentication ([#5013](https://github.com/kedacore/keda/issues/5013))
```

Comment:
Note: If this PR passes review and will be included in 2.13, then this needs to be moved below to 2.13 section of changelog.

Reply:
previous checks succeeded. I've move to 2.13 section changelog within last commit.